Src/WitsmlExplorer.Frontend/.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_WE_URL=https://localhost/witsmlexplorer
+NEXT_PUBLIC_WE_API_URL=https://localhost/witsmlexplorer.api
```

Comment:
Using nextjs as the framework for the frontend application should remain as an implementation detail, and I don't see the need to expose that in the environment variable name. Could these be `WE_FRONTEND_URL` and `WE_API_URL` instead? (or use full name like `WITSMLEXPLORER_FRONTEND_URL` / `WITSMLEXPLORER_API_URL`)

Reply:
Incorporated the review comments and updated the related documentation